client/apollo/react/src/Heading/__tests__/HeadingWithSubheadings.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from "@testing-library/react";
+import { HeadingWithSubheadings } from "../HeadingWithSubheadings";
+
+describe("TitleWithSubtitles", () => {
+  it("should render title in h1 and the two subtitles", () => {
+    const title = "Sample Title";
+    const firstSubtitle = "First Subtitle";
+    const secondSubtitle = "Second Subtitle";
+
+    render(
+      <HeadingWithSubheadings
+        title={title}
+        firstSubtitle={firstSubtitle}
+        secondSubtitle={secondSubtitle}
+      />,
+    );
+
+    screen.getByRole("heading", { name: title, level: 1 });
+    expect(screen.getByText(firstSubtitle)).toBeInTheDocument();
+    expect(screen.getByText(secondSubtitle)).toBeInTheDocument();
+  });
+
+  it("should render title in h3 and the subtitle", () => {
+    const title = "Sample Title";
+    const firstSubtitle = "First Subtitle";
+
+    render(
+      <HeadingWithSubheadings
+        title={title}
+        firstSubtitle={firstSubtitle}
+        level={3}
+      />,
+    );
+
+    screen.getByRole("heading", { name: title, level: 3 });
```

Comment:
il manque un `expect` ici, tu pourrais faire un truc du genre:
```suggestion
    const heading = screen.getByRole("heading", { name: title });
    expect(heading.tagName).toStrictEqual('H2');
```
comme cela tu vérifie bien que ton `heading` et bien un `h3`